packages/ui-components/stories/story-utils/api/data.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import faker from 'faker';
+
+const id = faker.random.uuid;
+
+export const ENTITIES = [
+  // projects
+  { id: id(), code: 'covidau', name: 'COVID-19 Australia' },
+  {
+    id: id(),
+    code: 'europe',
+    name: 'European Project',
+  },
+  { id: id(), code: 'empty', name: 'Empty hierarchy' },
+
+  // Australia
+  { id: id(), code: 'AU', name: 'Australia' },
+  { id: id(), code: 'AU_New_South_Wales', name: 'New South Wales' },
+  { id: id(), code: 'AU_New_South_Wales_Albury', name: 'Albury' },
+  { id: id(), code: 'AU_New_South_Wales_Byron', name: 'Byron' },
+  { id: id(), code: 'AU_New_South_Wales_Sydney', name: 'Sydney' },
+  { id: id(), code: 'AU_Queensland', name: 'Queensland' },
+  { id: id(), code: 'AU_Queensland_Brisbane', name: 'Brisbane' },
+  { id: id(), code: 'AU_Queensland_Cairns', name: 'Cairns' },
+  { id: id(), code: 'AU_Tasmania', name: 'Tasmania' },
+  { id: id(), code: 'AU_Victoria', name: 'Victoria' },
+  { id: id(), code: 'AU_Victoria_Ballarat', name: 'Ballarat' },
+  { id: id(), code: 'AU_Victoria_Melbourne', name: 'Melbourne' },
+  { id: id(), code: 'AU_Western_Australia', name: 'Western Australia' },
+
+  // Austria
+  { id: id(), code: 'AT', name: 'Austria' },
```

Comment:
I thought a bit about using `faker` for all this data, but I realised that it returns random cities for a selected country.

Seeing Athens inside Austria (this time - could be Barcelona next time) would be confusing. We wouldn't be able to tell if this works or not :smile: 

This also tests the column overflow logic as the list of European countries is long